scripts/shared/build_image.sh

```diff
@@ -47,7 +47,7 @@ if [[ "$cache" = true ]]; then
 fi
 
 # Rebuild the image to update any changed layers and tag it back so it will be used.
-buildargs_flag='--build-arg BUILDKIT_INLINE_CACHE=1'
+buildargs_flag="--build-arg BUILDKIT_INLINE_CACHE=1 --build-arg CUTTING_EDGE=${CUTTING_EDGE}"
```

Comment:
No no please no specific code, that's what `buildargs` is for exactly.

Set it in the `release.yml` GHA file

Reply:
but what about building it locally then? :?

Reply:
I mean:

git checkout release-0.8
make images